.github/workflows/build-tests.yml

```diff
@@ -0,0 +1,87 @@
+name: Build Tests
+
+on: [push, pull_request]
+
+jobs:
+  init:
+    name: Initializing Workflow
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.init.outputs.matrix }}
+      repo: ${{ steps.init.outputs.repo }}
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Initialize workflow
+        id: init
+        env:
+          BASE64_MATRIX: ${{ secrets.BASE64_MATRIX }}
+          BASE64_REPO: ${{ secrets.BASE64_REPO }}
+        run: |
+          tests/bin/init-workflow.sh
+
+  build-test:
+    name: Build Test
+    needs: init
+    runs-on: ubuntu-latest
+    strategy:
+      matrix: ${{ fromJSON(needs.init.outputs.matrix) }}
+    container: fedora:${{ matrix.os }}
+    steps:
+    - name: Clone repository
+      uses: actions/checkout@v2
+
+    - name: Install dependencies
+      run: |
+        dnf install -y dnf-plugins-core maven
+        dnf copr enable -y ${{ needs.init.outputs.repo }}
+        dnf builddep -y --spec ldapjdk.spec
+
+    - name: Build LDAP JDK with Ant
+      run: |
+        ./build.sh
+
+    - name: Install JSS into Maven repo
+      run: |
+        mvn install:install-file \
+            -Dfile=/usr/lib/java/jss.jar \
+            -DgroupId=org.dogtagpki \
+            -DartifactId=jss \
+            -Dversion=5.3.0-SNAPSHOT \
```

Comment:
Remembering to update this is going to get annoying isn't it.

Do you think it is worth pulling this from `pom.xml` somehow? I took a quick look to see if `xmllint` lets you do it easily, and it looks like a minefield.

Reply:
I think if we forget to update it the CI will fail, so there's an automatic reminder, but yeah, it is annoying. I haven't figured out an easy way to do it, but feel free to experiment.